packages/alert-banner/index.tsx

```diff
@@ -12,6 +12,11 @@ import fragment from './fragment.graphql'
 import s from './style.module.css'
 import analytics from './analytics'
 
+interface AlertBannerCSSVariables {
+  '--background-color'?: string
+  '--color'?: string
+}
```

Comment:
I like this a lot. Maybe add comments above explaining when they _should_ be used to hopefully hedge against "mis-use" (e.g. customizing every single time we render)?